core/src/main/java/io/questdb/cutlass/text/CairoTextWriter.java

```diff
@@ -358,7 +358,7 @@ void prepareTable(
                 }
                 break;
             default:
-                throw CairoException.instance(0).put("name is reserved [table=").put(tableName).put(']');
+                throw CairoException.critical(0).put("name is reserved [table=").put(tableName).put(']');
```

Comment:
non-critical

Reply:
Let me address this one a bit later.

Reply:
Addressed in 345b95b650003b2547353cf524ea8f87e2adee51